srcpkgs/mac-fdisk/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'mac-fdisk'
+pkgname=mac-fdisk
+version=0.1
+revision=1
+only_for_archs="i686 i686-musl x86_64 x86_64-musl ppc ppc-musl ppc64-musl ppc64le ppc64le-musl"
```

Comment:
is there a reason why this can't be build for the remaining arches?

Reply:
Yes, it fails to build on arm and. Would it be better to mark it as broken on arm?

Reply:
yes